src/pyconverter/xml2py/custom_functions.py

```diff
@@ -48,39 +47,53 @@ def get_docstring_lists(filename: str) -> Tuple[list[str], list[str], list[str],
     """
     pyfile = open(filename, "r")
     lines = pyfile.readlines()
+    pyfile.close()
```

Comment:
```suggestion
    lines = None
    with open(filename, "r") as pyfile:
        lines = pyfile.readlines()
```

Use the context manager - much safer